o2.sh

```diff
@@ -57,6 +57,17 @@ if [[ $GIT_TAG == master ]]; then
 fi
 make ${CONTINUE_ON_ERROR+-k} ${JOBS+-j $JOBS} install
 
+DEVEL_SOURCES="`readlink $SOURCEDIR || echo $SOURCEDIR`"
+# This really means we are in development mode. We need to make sure we
+# use the real path for sources in this case. We also copy the
+# compile_commands.json file so that IDEs can make use of it directly, this
+# is a departure from our "no changes in sourcecode" policy, but for a good reason
+# and in any case the file is in gitignore.
+if [ "$DEVEL_SOURCES" != "$SOURCEDIR" ]; then
+  perl -p -i -e "s|$SOURCEDIR|$DEVEL_SOURCES|" compile_commands.json
+  ln -sf $BUILDDIR/compile_commands.json $DEVEL_SOURCES/compile_commands.json
+fi
```

Comment:
I am bit confused here since you don't actually seem to produce `compile_commands.json` using the "CMAKE_EXPORT_COMPILE_COMMANDS" option.